libs/sql-schema-describer/tests/postgres/mod.rs

```diff
@@ -19,10 +19,7 @@ pub async fn get_postgres_describer(sql: &str, db_name: &str) -> postgres::SqlSc
         .unwrap();
 
     let drop_schema = format!("DROP SCHEMA IF EXISTS \"{}\" CASCADE;", SCHEMA);
-    client
-        .query_raw(drop_schema.as_str(), &[])
-        .await
-        .expect("dropping schema");
+    client.query_raw(&drop_schema, &[]).await.expect("dropping schema");
```

Comment:
Nitpick: for DDL statements, to avoid them being prepared, the recommended quaint method is `client.raw_cmd()` now.

Reply:
Will do